config/remotePlugin.js

```diff
@@ -49,6 +49,17 @@ const flagExtensions = [
 ];
 
 const contextProviderExtensions = [
+  {
+    type: 'core.context-provider',
+    properties: {
+      provider: {
+        $codeRef: 'WorkspaceContext.WorkspaceProvider',
+      },
+      useValueHook: {
+        $codeRef: 'WorkspaceContext.useActiveWorkspace',
+      },
+    },
+  },
   {
     type: 'core.context-provider',
```

Comment:
Do we need this context provider now? We can just merge this one into workspace context provider and set namespace based on what is returned in the lister API.